### ISSUE TITLE
Poor visualizations, getting zero SDR, SIR, etc. on evaluation

### ISSUE THREAD

**Issue by deepakee13** [hangzhaomit/Sound-of-Pixels#7]
I was trying to evaluate on 16 videos using downloaded trained model but I am unable to see the results in visualization. Video1 and video2 have only 3 frames each with no audio and predicted audio are also silent.

I'm getting the following output after evaluation:

**Loading weights for net_frame
Loading weights for net_synthesizer
samples: 6300
samples: 16
1 Epoch = 196 iters
Evaluating at 0 epochs...
[Eval] iter 0, loss: 0.0115
[Eval Summary] Epoch: 0, Loss: 0.0115, SDR_mixture: 0.0000, SDR: 0.0000, SIR: 0.0000, SAR: 0.0000
Plotting html for visualization...
Evaluation Done!**

Hope I would get some help
Thanks

**Comment by deepakee13** [comment]
Problem got solved

**Comment by yifan254** [comment]
hi, deepakee13,  I got the same problem as you, so can you give me some ideas on how to solve this problem? thanks~

**Comment by deepakee13** [comment]
I removed normalization coefficient of 2^-31 from raw audio and took
transpose of it which was probably in base.py and then it worked fine.

........................
Deepak Kumar

**Comment by JusperLee** [comment]
hello, I want to know what can be calculated by modifying that part in base.py.

**Comment by deepakee13** [comment]
See, In my case, I was using pretrained model and validating the dataset on it. But as I said above in the original code inside the function _load_audio_file(), they were normalizing by a factor of 2^-31, which in my case made raw audio to zero, by removing that part, i was able to get some result and i also recollect that i had to take transpose of raw audio while converting to mono.
Otherwise you can remove the If loop for handling the .mp3 format and use librosa or other library to load the audio file.

**Comment by JusperLee** [comment]
I modified the code to make it work, and then I got very unsatisfactory results.  The separation effect is also relatively poor.

**Comment by deepakee13** [comment]
I saw your commit in the code, I did the same. but i used pretrained model and result was good enough in case of validation on MUSIC dataset videos ( the reason might be the model are already trained on those videos) . But I was also getting very poor result on inference.

**Comment by JusperLee** [comment]
I did not use a pre-trained model.  I trained from scratch and found that the results of the training were not very good.

**Comment by parth1497** [comment]
@deepakee13 can you share the your  code snippet after you made the correction to the function? (The transpose part specifically) Thanks!

**Comment by parth1497** [comment]
@JusperLee If you can also provide any input then it will be great. Thanks in advance.

**Comment by JusperLee** [comment]
@parth1497 #9
